core/services/offchainreporting/delegate.go

```diff
@@ -193,7 +193,7 @@ func (d Delegate) ServicesForSpec(jobSpec job.Job) (services []job.Service, err
 		runResults := make(chan pipeline.RunWithResults, d.config.JobPipelineResultWriteQueueDepth())
 		oracle, err := ocr.NewOracle(ocr.OracleArgs{
 			Database: ocrdb,
-			Datasource: dataSource{
+			Datasource: &dataSource{
```

Comment:
Curious, why change this to a pointer?

Reply:
because of this new write https://github.com/smartcontractkit/chainlink/pull/4121/files#diff-8c4ea9636fec98879a197b828c064da206d788de90a0c6754b413dbe6f0891cbR85